cpp/src/copying/pack.cpp

```diff
@@ -182,14 +182,11 @@ class metadata_builder_impl {
 
   std::vector<uint8_t> build() const
   {
-    // convert to anonymous bytes
-    std::vector<uint8_t> metadata_bytes;
-    auto const metadata_begin = reinterpret_cast<uint8_t const*>(metadata.data());
-    std::copy(metadata_begin,
-              metadata_begin + (metadata.size() * sizeof(detail::serialized_column)),
-              std::back_inserter(metadata_bytes));
-
-    return metadata_bytes;
+    auto output = std::vector<uint8_t>(metadata.size() * sizeof(detail::serialized_column));
+    std::memcpy(reinterpret_cast<void*>(output.data()),
+                reinterpret_cast<void const*>(metadata.data()),
```

Comment:
I don't think the `reinterpret_cast` is necessary
```suggestion
    std::memcpy(output.data(), metadata.data(),
```

Reply:
Thanks.